cddl/contrib/opensolaris/tests/oclo/oclo_errors.c

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
The paths of files under cddl/contrib/opensolaris should follow upstream's layout, which in this case means that these should live in cddl/contrib/opensolaris/tests/os-tests/tests/oclo.

Reply:
Fixed.